### PR TITLE
ES-850323 - Add FirstSlideNumber API code snippets in UG

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Add **FirstSlideNumber** API code snippets in UG

TaskLink: <a href="https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/850323">Add FirstSlideNumber API code snippets in UG</a>

Regards,
Venkatesh

### CODE REVIEW

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 1]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
+
+## First slide number
+
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:488]: Change the first slide number of PowerPoint presentation using **FirstSlideNumber** API.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 2]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+{
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Get the FirstSlideNumber of Presentation.
+        int firstSlideNumber = pptxDoc.FirstSlideNumber;
+
+        //Modify the value for FirstSlideNumber.
+        pptxDoc.FirstSlideNumber = 10;
+
+        //Save the PowerPoint Presentation as stream
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:507]: Cross check the xml comment

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 3]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+    pptxDoc.FirstSlideNumber = 10
+
+    'Save the PowerPoint Presentation as stream.
+    pptxDoc.Save("Result.pptx")
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+N> First slide number is the starting slide number of presentation, and this API allows to set the first slide number from 0 to 9999.
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:547]: Get content review

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/Presentation/Working-with-PowerPoint-presentation.md:488]: modified

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/Presentation/Working-with-PowerPoint-presentation.md:507]: Resolved

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 4]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
+
+## First slide number
+
+Change the first slide number of PowerPoint presentation using **FirstSlideNumber** API.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:489]: Change the first slide number of the PowerPoint presentation using the **FirstSlideNumber** API.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 5]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
\ No newline at end of file
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PowerPoint-Presentation/Mark-as-final).
+
+## First slide number
+
+Change the first slide number of PowerPoint presentation using **FirstSlideNumber** API. 
+
+The following code snippet illustrates how to modify the first slide number in the PowerPoint presentation.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:491]: The following code sample illustrates how to modify the first slide number in the PowerPoint presentation.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 6]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+
+//Open an existing PowerPoint Presentation.
+using (FileStream inputStream = new FileStream("Data/Input.pptx", FileMode.Open))
+{
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Get the FirstSlideNumber of Presentation.
+        int firstSlideNumber = pptxDoc.FirstSlideNumber;
+
+        //Modify the value for FirstSlideNumber.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:504]: //Modify the value for the FirstSlideNumber.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 7]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+        pptxDoc.FirstSlideNumber = 10;
+
+        //Save the PowerPoint Presentation as stream
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:507]: //Save the PowerPoint Presentation as a stream.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 8]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+using (IPresentation pptxDoc = Presentation.Open("Data/Input.pptx"))
+{
+    //Get the FirstSlideNumber of Presentation.
+    int firstSlideNumber = pptxDoc.FirstSlideNumber;
+
+    //Modify the value for FirstSlideNumber.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:523]: //Modify the value for the FirstSlideNumber.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 9]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+}
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+Using pptxDoc As IPresentation = Presentation.Open("Data/Input.pptx")
+    'Get the FirstSlideNumber of Presentation.
+    Dim firstSlideNumber As Integer = pptxDoc.FirstSlideNumber
+
+    'Modify the value for FirstSlideNumber.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:537]: 'Modify the value for the FirstSlideNumber.

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 10]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+    pptxDoc.FirstSlideNumber = 10
+
+    'Save the PowerPoint Presentation.
+    pptxDoc.Save("Result.pptx")
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+N> First slide number is the starting slide number of presentation, and this API allows to set the first slide number from 0 to 9999.
```
Comment (Chrispineagunja) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:547]: N> The first slide number is the starting slide number of the presentation, and this API allows you to set the first slide number from 0 to 9999.

**Review by Chrispineagunja — COMMENTED** [review]
Content reviewed

File: File-Formats/Presentation/Working-with-PowerPoint-presentation.md [hunk 11]
```diff
@@ -482,4 +482,68 @@ pptxDoc.Close()
...
+{
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Get the FirstSlideNumber of Presentation.
+        int firstSlideNumber = pptxDoc.FirstSlideNumber;
+
+        //Modify the value for the FirstSlideNumber.
+        pptxDoc.FirstSlideNumber = 10;
+
+        //Save the PowerPoint Presentation as a stream.
```
Comment (MohanaselvamJothi) [inline File-Formats/Presentation/Working-with-PowerPoint-presentation.md:507]: //Save the PowerPoint Presentation

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**